### PR TITLE
[docs] Final editorial changes embeddings docs.

### PR DESCRIPTION
**Pull request by pablo-gar**
- Typo corrections.
- Adjusted language as suggested by internal directives

### CODE REVIEW

File: api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb [hunk 1]
```diff
@@ -182,10 +182,11 @@
     "\n",
     "### Preparaing single-cell data\n",
     "\n",
-    "Let's load the test data. In preparation to use with Geneformer the following: \n",
-    "- Sets the index as the ENSEMBL ID and stores it in the `obs` column `\"ensembl_id\"`\n",
-    "- Adds read counts to the `obs` column `\"n_counts\"`\n",
-    "- Adds and ID column to be used for joining later in the  `obs` column `\"joinid\"`\n",
+    "Let's load the test data. In preparation to use with Geneformer to do the following: \n",
```
Comment (bkmartinjr) [inline api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb:185]: `In preparation to use with Geneformer to do the following` does not scan - needs an edit

**Review by bkmartinjr — COMMENTED** [review]
CI is failing

File: api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb [hunk 2]
```diff
@@ -182,10 +182,11 @@
     "\n",
     "### Preparaing single-cell data\n",
```
Comment (bkmartinjr) [inline api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb:183]: typo in first word

**Review by bkmartinjr — APPROVED** [review]
Other than one remaining typo, noted inline, lgtm.

File: api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb [hunk 3]
```diff
@@ -7,7 +7,7 @@
    "source": [
     "# Geneformer for cell class prediction and data projection\n",
     "\n",
-    "This notebook provides examples to utilize the CELLxGENE-maintained fine-tuned Geneformer model with user data. For more information on the model please refer to the [Census model page](https://cellxgene.cziscience.com/census-models).\n",
+    "This notebook provides examples to utilize the CELLxGENE Collaboration fine-tuned Geneformer model with user data. For more information on the model please refer to the [Census model page](https://cellxgene.cziscience.com/census-models).\n",
```
Comment (atarashansky) [inline api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb:10]: What is "CELLxGENE Collaboration"? is that the name of this initiative? Or is Collaboration meant to be lowercased?

File: api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb [hunk 4]
```diff
@@ -182,10 +182,11 @@
...
     "\n",
-    "Let's load the test data. In preparation to use with Geneformer the following: \n",
-    "- Sets the index as the ENSEMBL ID and stores it in the `obs` column `\"ensembl_id\"`\n",
-    "- Adds read counts to the `obs` column `\"n_counts\"`\n",
-    "- Adds and ID column to be used for joining later in the  `obs` column `\"joinid\"`\n",
+    "Let's load the test data. In preparation to use with Geneformer we do the following: \n",
+    "\n",
+    "- Set the index as the ENSEMBL ID and stores it in the `obs` column `\"ensembl_id\"`\n",
+    "- Add read counts to the `obs` column `\"n_counts\"`\n",
+    "- Add and ID column to be used for joining later in the  `obs` column `\"joinid\"`\n",
```
Comment (atarashansky) [inline api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb:189]: Add an* ID column

File: api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb [hunk 5]
```diff
@@ -484,7 +485,7 @@
    "id": "e9a80b25-c9a5-49e5-9609-a0f90ac2d107",
    "metadata": {},
    "source": [
-    "### Performing inference of cell class"
+    "### Performing inference of cell subclass"
```
Comment (atarashansky) [inline api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb:488]: Is the meaning of "subclass" as opposed to "class/type" defined anywhere in the notebook?

**Review by atarashansky — COMMENTED** [review]
"CELLxGENE Collaboration" sounds quite generic. Wish there were a better name for it.

File: api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb (hunk 3 above)
Comment (pablo-gar) [reply api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb:10]: We had to change the name of this Tier per internal directives.

File: api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb (hunk 4 above)
Comment (pablo-gar) [reply api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb]: Thanks

File: api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb (hunk 5 above)
Comment (pablo-gar) [reply api/python/notebooks/analysis_demo/comp_bio_geneformer_prediction.ipynb:490]: I added a description in top-level description. Thanks

**Pull request merged**